src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -352,6 +355,29 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:simpleType name="cookie_samesite">
+        <xsd:restriction base="xsd:string">
+            <xsd:enumeration value="" />
+            <xsd:enumeration value="lax" />
+            <xsd:enumeration value="strict" />
+        </xsd:restriction>
+    </xsd:simpleType>
+
+    <xsd:simpleType name="sid_bits_per_character">
+        <xsd:restriction base="xsd:positiveInteger">
+            <xsd:enumeration value="4" />
+            <xsd:enumeration value="5" />
+            <xsd:enumeration value="6" />
+        </xsd:restriction>
+    </xsd:simpleType>
+
+    <xsd:simpleType name="sid_length">
+        <xsd:restriction base="xsd:positiveInteger">
+            <xsd:minInclusive value="22" />
+            <xsd:maxExclusive value="256" />
```

Comment:
If `maxEclusive` means that `256` is forbidden ... I'd like to ask you to reconsider it:

* PHP manual says: *"Session ID length can be between 22 to 256."* so 256 seems valid. http://php.net/manual/en/session.configuration.php#ini.session.sid-length
* PHP source code defines it here: https://github.com/php/php-src/blob/623911f993f39ebbe75abe2771fc89faf6b15b9b/ext/session/session.c#L80 and checks it here: https://github.com/php/php-src/blob/623911f993f39ebbe75abe2771fc89faf6b15b9b/ext/session/session.c#L355 and here: https://github.com/php/php-src/blob/623911f993f39ebbe75abe2771fc89faf6b15b9b/ext/session/session.c#L727  so it looks like `256` is allowed

Reply:
Thanks, I missed that point, its inclusive of course, I'll correct that soon

Reply:
fixed, thank you @javiereguiluz